nebula/mocks/Mozilla/VPN/MZFilterProxyModel.qml

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+
+Item {
+  property var filterCallback: () => {}
```

Comment:
This should not required. Why do we need to lock this object? Is it for qmllint?

Reply:
Yes, this was an attempt to clean up some qmllint warnings. It was complaining that it couldn't determine the types of properties in an `MZFilterProxyModel`, so I simply attempted to add a mocked version. Unfortunately, there are a bunch of other QML linter warnings that are going to much harder to resolve. I'm debating if I should just back out of the linter work and deal with it in another PR.

Reply:
@strseb  is at work on a much better solution with QML modules. So I am going to revert this.